README.md

```diff
@@ -44,6 +44,21 @@ Once installed, add the following to vscode's `settings.json` file:
   },
 ```
 
+### Sublime Text
+
+Install the [rubyfmt plugin](https://github.com/toreriklinnerud/sublime-rubyfmt/) from [Package Control](https://packagecontrol.io): Install Package -> rubyfmt. 
+
+`ruby` and `rubyfmt` are assumed to be on path. Ruby files are formatted on save or by pressing `Alt + ;` or on macOS: `Cmd + ;`
```

Comment:
why does ruby need to be on the path?

Reply:
I had the problem where rubyfmt would crash when there were syntax errors in the ruby file so if rubyfmt exits with an error I'm invoking `ruby -c` to check for syntax errors and surface them in the editor window.